cmd/subnetcmd/deploy.go

```diff
@@ -119,14 +119,31 @@ func deploySubnet(cmd *cobra.Command, args []string) error {
 	case models.Local:
 		app.Log.Debug("Deploy local")
 		deployer := subnet.NewLocalSubnetDeployer(app)
-		if err := deployer.DeployToLocalNetwork(chain, chainGenesis); err != nil {
+		subnetID, blockchainID, err := deployer.DeployToLocalNetwork(chain, chainGenesis)
+		if err != nil {
 			if deployer.BackendStartedHere() {
 				if innerErr := binutils.KillgRPCServerProcess(app); innerErr != nil {
 					app.Log.Warn("tried to kill the gRPC server process but it failed: %w", innerErr)
 				}
 			}
+			return err
 		}
-		return err
+		sc, err := app.LoadSidecar(chain)
```

Comment:
maybe you want to first get sidecar before modifying the blockchain

Reply:
ok